dnn/variational_with_real_data.ipynb

```diff
@@ -7,23 +7,23 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.        abs_var = keras.layers.Lambda(lambda t: tf.abs(t))(var_output)
</pre><p>Get rid of <code>abs()</code> as discussed, since we have a smooth way to do this now as of our most recent pairing session. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Ack, updated.